apps/react-app/project.json

```diff
@@ -4,12 +4,12 @@
   "targets": {
     "watch": {
       "options": {
-        "commands": ["vite"]
+        "commands": ["bunx vite"]
       }
     },
     "build": {
       "options": {
-        "commands": ["tsc", "vite build"]
+        "commands": ["bunx tsc", "bunx vite build"]
```

Comment:
using bunx vite will basically run vite in node and not bun. Same with TSC here. You need force bun with --bun
`
``suggestion
        "commands": ["bunx --bun tsc", "bunx --bun vite build"]
```

bun also has bulit in support for typescript compling so you dont need to run TSC. Whats the reasoning for it?

Reply:
done!  thx!

Reply:
@Jordan-Hall regarding:

"bun also has bulit in support for typescript compling so you dont need to run TSC. Whats the reasoning for it?"

I want to publish reusable libraries for others from some of my projects.  Thus I want to transpire to *.js that is non-bundled, non-minified for easy reuse, tree shaking, etc.

I guess you'd recommend that I use https://bun.sh/docs/bundler?  I guess that can be used for libraries and not just end user React apps?

Reply:
Correct working on getting this to work on nx-bun but you can use cli for it


bun build ./index.tsx --outdir ./out